src/nvim/globals.h

```diff
@@ -55,6 +55,51 @@
 # endif
 #endif
 
+#ifndef FILETYPE_FILE
+# define FILETYPE_FILE  "filetype.vim"
+#endif
+
+#ifndef FTPLUGIN_FILE
+# define FTPLUGIN_FILE  "ftplugin.vim"
+#endif
+
+#ifndef INDENT_FILE
+# define INDENT_FILE    "indent.vim"
+#endif
+
+#ifndef FTOFF_FILE
+# define FTOFF_FILE     "ftoff.vim"
+#endif
+
+#ifndef FTPLUGOF_FILE
+# define FTPLUGOF_FILE  "ftplugof.vim"
+#endif
+
+#ifndef INDOFF_FILE
+# define INDOFF_FILE    "indoff.vim"
+#endif
+
+#define DFLT_ERRORFILE  "errors.err"
+
+#ifndef SYS_VIMRC_FILE
+# define SYS_VIMRC_FILE "$VIM" PATHSEPSTR "sysinit.vim"
+#endif
+
+#ifndef DFLT_HELPFILE
+# define DFLT_HELPFILE  "$VIMRUNTIME" PATHSEPSTR "doc" PATHSEPSTR "help.txt"
+#endif
+
+#ifndef SYNTAX_FNAME
+# define SYNTAX_FNAME   "$VIMRUNTIME" PATHSEPSTR "syntax" PATHSEPSTR "%s.vim"
```

Comment:
I am pretty sure this is *not* safe, because this will be `pseps`, and it cannot be something else because e.g. `add_pathsep` function needs to depend on `&shellslash` option.